### ISSUE TITLE
Release 12 | ROT/SP release requests

### ISSUE THREAD

**Issue by morlandi7** [oxidecomputer/hubris#1725]
Please link any changes requiring an ROT/SP release to be included in the next customer release

```[tasklist]
### ROT/SP Updates
- [ ] Add a draft title or issue reference here

**Comment by labbott** [comment]
https://github.com/oxidecomputer/hubris/pull/1934 this should be in the PSC builds

**Comment by Aaron-Hartwig** [comment]
yes both https://github.com/oxidecomputer/quartz/issues/246 and https://github.com/oxidecomputer/quartz/issues/247 will need to be rolled into a hubris commit to release for Sidecar. I will update this ticket with that PR once I have it.

**Comment by Aaron-Hartwig** [comment]
The Hubris PR with the Quartz fixes is https://github.com/oxidecomputer/hubris/pull/1942

**Comment by morlandi7** [comment]
release complete
